wikipron/scrape.py

```diff
@@ -48,7 +48,8 @@ def _scrape_once(data, config: Config) -> Iterator[Pair]:
             if " " in pron:
                 continue
             pron = config.process_pron(pron)
-            yield (word, pron)
+            if pron:
+                yield (word, pron)
```

Comment:
I had to implement the "pron could be `None`" logic inside `config.process_pron` for testability, or else it's hard, if not impossible, to actually test the bug fix by making `unittest.mock` work with various network API calls. The codebase as-is has the test suite outside of the library code, plus we just so happen to have a name clash between the `wikipron.scrape` and the module `scrape.py`...